src/main/java/com/leshchenko/springboot/service/impl/ReviewParserFromCsvFileImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.leshchenko.springboot.service.impl;
+
+import com.leshchenko.springboot.model.Review;
+import com.leshchenko.springboot.service.FileReaderService;
+import com.leshchenko.springboot.service.ReviewParserFromCsvRecord;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReviewParserFromCsvFileImpl implements FileReaderService<List<Review>> {
```

Comment:
```suggestion
public class ReviewParserFromCsvFileImpl implements FileReaderService<Review> {
```
Можемо так відрефакторити? А в методі `public List<Review> getContent(String path)` все залишити як є

Reply:
changed interface and method getContent generic from <T> to <list<T>>